examples/apps/spaces/src/fluid-object/spaces.ts

```diff
@@ -50,8 +46,14 @@ export interface ISpacesItem {
 /**
  * Spaces is the main component, which composes a SpacesToolbar with a SpacesStorage.
  */
-export class Spaces extends DataObject implements IFluidHTMLView {
-    private storageComponent: SpacesStorage | undefined;
+export class Spaces extends DataObject {
+    private _storageComponent: SpacesStorage | undefined;
+    public get storageComponent(): SpacesStorage {
```

Comment:
Given that `Spaces` now represents data only, what are some of the advantageous exposing `SpacesStorage` ? Is it to contain the scope of FF changes that can trigger re-rendering?

Reply:
Good question - it would probably be reasonable to merge the two now for simplicity, I originally split them apart (#2108) because another demo was reaching through a `Spaces` to get at its data model, so I split that data model out for reuse.  Now that the other demo is gone the split is not particularly valuable.

Reply:
I'm curious what would be some of the underlining reasons/advantageous to split the data model across multiple dataobjects/datastores - but that is another topic.

Reply:
In that prior PR, the best explanation of the value would be something like "separate shipping channels" -- a more-realistic construction would have split the storage out to a separate shared package that was imported by both the Spaces and waterpark packages.